Makefile

```diff
@@ -3,6 +3,7 @@ PACKAGE_NAME = image-builder
 .PHONY: build
 build:
 	go build -o image-builder ./cmd/image-builder/
+	go build -o image-builder-migrate-db ./cmd/image-builder-migrate-db/
```

Comment:
We'd need to run this whenever there's a new version, but we don't want to run this within `image-builder` as then multiple instances would migrate the database all at once. Which I'm sure wouldn't break, or at least it shouldn't, but it's better to run it as a job, once, with every (re)deployment.